stdcommands/general_commands.go

```diff
@@ -533,7 +533,7 @@ func HandleMessageCreate(evt *eventsystem.EventData) {
 	taken := time.Duration(time.Now().UnixNano() - parsed)
 
 	started := time.Now()
-	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "Gatway (http send -> gateway receive time): "+taken.String())
+	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "Gateway (http send -> gateway receive time): "+taken.String())
 	httpPing := time.Since(started)
 
 	common.BotSession.ChannelMessageEdit(m.ChannelID, m.ID, "HTTP API (Edit Msg): "+httpPing.String()+"\nGatway: "+taken.String())
```

Comment:
The same typo is down here aswell if you wanna fix that

Reply:
Hmmm, I was sure I fixed that first lol... Everything is fixed now. :)